sdk-base/build.gradle.kts

```diff
@@ -17,14 +17,24 @@ android {
     testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
   }
 }
+val buildFromSource: String by project
 
 dependencies {
   implementation(Dependencies.kotlin)
   implementation(Dependencies.mapboxBase)
   implementation(Dependencies.androidxAnnotations)
   api(Dependencies.mapboxGestures)
-  api(Dependencies.mapboxGlNative)
-  api(Dependencies.mapboxCoreCommon)
+  if (buildFromSource.toBoolean()) {
+    api(project(":maps-core"))
+  } else {
+    api(Dependencies.mapboxGlNative)
+  }
+
+  if (buildFromSource.toBoolean()) {
+    api(project(":common"))
+  } else {
+    api(Dependencies.mapboxCoreCommon)
+  }
```

Comment:
let's merge it 😄 
```
if (buildFromSource.toBoolean()) {
  api(project(":maps-core"))
  api(project(":common"))
} else {
  api(Dependencies.mapboxGlNative)
  api(Dependencies.mapboxCoreCommon)
}
```
or use separate flags to build common from source separately from gl native